package.json

```diff
@@ -62,17 +67,14 @@
     "jest-stare": "^2.5.1",
     "js-yaml": "^4.1.0",
     "jsonfile": "^6.0.0",
-    "lerna": "^5.4.3",
-    "madge": "^4.0.1",
+    "madge": "^6.1.0",
     "mustache": "^4.2.0",
     "npm-lockfile": "^3.0.7",
     "rimraf": "^5.0.0",
-    "shebang-regex": "^2.0.0",
     "symlink-dir": "^5.2.1",
-    "syncpack": "^5.6.10",
     "ts-jest": "^29.1.2",
     "ts-node": "^7.0.1",
-    "typedoc": "^0.23.20",
+    "typedoc": "^0.25.8",
```

Comment:
didn't they changed the output format and if we want to keep the old format we have to use the `--categorizeByGroup` option?

Reply:
Discussed at my desk, the new format looks better.